tests/e2e/e2e_workflow.spec.js

```diff
@@ -56,19 +67,17 @@ describe('Basic end-to-end Workflow', function () {
     });
   });
 
-  it('Perform `ng test`', function(done) {
-    this.timeout(30000);
+  it('Perform `ng test` after initial build', function() {
+    this.timeout(300000);
 
-    return ng([
-      'test'
-    ]).then(function(err) {
-      // TODO when `ng test` will be implemented
-      //expect(err).to.be.equal(1);
-      done();
+    return ng(testArgs)
+    .then(function(result) {
+      expect(result.exitCode).to.be.equal(0);
```

Comment:
for karma tests yes, based on their API it is how we can discern if our tests passed/failed via our CLI